packages/push/src/genome/plushy.rs

```diff
@@ -72,63 +138,66 @@ mod test {
         }
         .generate(&mut rng)
         .unwrap();
-        assert_eq!(10, plushy.instructions.len());
+        assert_eq!(10, plushy.genes.len());
     }
 
     #[test]
     fn umad() {
         let mut rng = thread_rng();
 
-        let instruction_options = [PushInstruction::InputVar(VariableName::from("x"))];
+        let instruction_options = [PushGene::Instruction(PushInstruction::InputVar(
+            VariableName::from("x"),
+        ))];
         let umad = Umad::new(0.3, 0.3, instruction_options);
 
-        let parent_instructions = vec![
-            PushInstruction::IntInstruction(IntInstruction::Add),
-            PushInstruction::BoolInstruction(BoolInstruction::And),
-            PushInstruction::BoolInstruction(BoolInstruction::Or),
-            PushInstruction::IntInstruction(IntInstruction::Multiply),
-            PushInstruction::IntInstruction(IntInstruction::Add),
-            PushInstruction::BoolInstruction(BoolInstruction::And),
-            PushInstruction::BoolInstruction(BoolInstruction::Or),
-            PushInstruction::IntInstruction(IntInstruction::Multiply),
-            PushInstruction::IntInstruction(IntInstruction::Add),
-            PushInstruction::BoolInstruction(BoolInstruction::And),
-            PushInstruction::BoolInstruction(BoolInstruction::Or),
-            PushInstruction::IntInstruction(IntInstruction::Multiply),
-            PushInstruction::IntInstruction(IntInstruction::Add),
-            PushInstruction::BoolInstruction(BoolInstruction::And),
-            PushInstruction::BoolInstruction(BoolInstruction::Or),
-            PushInstruction::IntInstruction(IntInstruction::Multiply),
-            PushInstruction::IntInstruction(IntInstruction::Add),
-            PushInstruction::BoolInstruction(BoolInstruction::And),
-            PushInstruction::BoolInstruction(BoolInstruction::Or),
-            PushInstruction::IntInstruction(IntInstruction::Multiply),
+        let parent_genes: Vec<PushGene> = vec![
+            IntInstruction::Add.into(),
+            BoolInstruction::And.into(),
+            BoolInstruction::Or.into(),
+            IntInstruction::Multiply.into(),
+            IntInstruction::Add.into(),
+            BoolInstruction::And.into(),
+            BoolInstruction::Or.into(),
+            IntInstruction::Multiply.into(),
+            IntInstruction::Add.into(),
+            BoolInstruction::And.into(),
+            BoolInstruction::Or.into(),
+            IntInstruction::Multiply.into(),
+            IntInstruction::Add.into(),
+            BoolInstruction::And.into(),
+            BoolInstruction::Or.into(),
+            IntInstruction::Multiply.into(),
+            IntInstruction::Add.into(),
+            BoolInstruction::And.into(),
+            BoolInstruction::Or.into(),
+            IntInstruction::Multiply.into(),
         ];
         let parent = Plushy {
-            instructions: parent_instructions,
+            genes: parent_genes,
```

Comment:
```suggestion
            genes,
```

here this was missing from the suggestion above